examples/zkapps/anonymous-message-board/src/message.ts

```diff
@@ -0,0 +1,82 @@
+import {
+    Field,
+    SmartContract,
+    state,
+    State,
+    method,
+    PrivateKey,
+    PublicKey,
+    Poseidon,
+    Encoding,
+  } from 'o1js';
+  
+  export { Field, Encoding };
```

Comment:
Why are we exporting `Field` and `Encoding` imported from `o1js` here? We can remove that